src/status_im/ui/screens/wallet/ui/components/styles.cljs

```diff
@@ -0,0 +1,63 @@
+(ns status-im.ui.screens.wallet.ui.components.styles
+  (:require-macros [status-im.utils.styles :refer [defnstyle defstyle]])
```

Comment:
Why not put those in the main components ns? If they are wallet only it makes sense but if we can abstract it's better.
Also twice `ui` in ns looks funny.

Reply:
Anyway it could be just `status-im.ui.screens.wallet.components`, i don't see why we need ui again :)  Actually,  `status-im.screens.wallet.components` or `status-im.ui.wallet.components` could be even better imo

Reply:
agree